daemons/fenced/fenced_remote.c

```diff
@@ -2038,19 +2081,21 @@ process_remote_stonith_query(xmlNode * msg)
         peer = add_result(op, host, ndevices, dev);
     }
 
+    pcmk__set_result(&result, CRM_EX_OK, PCMK_EXEC_DONE, NULL);
+
     if (pcmk_is_set(op->call_options, st_opt_topology)) {
         /* If we start the fencing before all the topology results are in,
          * it is possible fencing levels will be skipped because of the missing
          * query results. */
         if (op->state == st_query && all_topology_devices_found(op)) {
             /* All the query results are in for the topology, start the fencing ops. */
             crm_trace("All topology devices found");
-            call_remote_stonith(op, peer, pcmk_ok);
+            request_peer_fencing(op, peer, &result);
```

Comment:
no result possible that needs resetting - right?

Reply:
correct, in this section result is always ok (no exit reason)